### PR TITLE
Feature audio player

### PR DESCRIPTION
**Pull request by gsmandur**
currently can't play back audio from s3 using the audio-player. But can play it back with the HTML audio element

### CODE REVIEW

File: client/src/components/ConversationDialog.js [hunk 1]
```diff
@@ -89,7 +90,8 @@ function ConversationDialog(props) {
         <DialogContent>
           <Container className={classes.root}>
             <Typography className={classes.title}>{title}</Typography>
-            <Paper className={classes.audio} />
+            <AudioPlayer audioURL={audioURL} />
+            <audio controls src={audioURL} />
```
Comment (gsmandur) [inline client/src/components/ConversationDialog.js]: Here I get an error with the AudioPlayer component when trying to playback the audio:
`Access to XMLHttpRequest at 'https://twitter-audio.s3.amazonaws.com/1564103805507' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

Comment (gsmandur) [reply client/src/components/ConversationDialog.js]: The audio can be played back fine using HTML5 <audio> tag

Comment (gsmandur) [reply client/src/components/ConversationDialog.js]: I read maybe you have to allow CORS on s3? I am not sure why this is happening

Comment (skassam21) [reply client/src/components/ConversationDialog.js]: You have to add CORS to the bucket:
https://docs.aws.amazon.com/AmazonS3/latest/user-guide/add-cors-configuration.html

**Comment by alaindavis90** [comment]
Ok. I’ll do that

**Comment by alaindavis90** [comment]
I have added the CORS configuration. The audio player is now functional. I
am going to merge this PR.

**Pull request merged**